src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -128,10 +128,16 @@ internal void LoadContext(ExecutionContext context, int initialPosition)
             LoadContext(context);
         }
 
-        public ExecutionContext LoadScript(Script script, CallFlags callFlags)
+        public ExecutionContext LoadScript(Script script, CallFlags callFlags, bool turingComplete = true)
         {
             ExecutionContext context = LoadScript(script);
-            context.GetState<ExecutionContextState>().CallFlags = callFlags;
+            ExecutionContextState state = context.GetState<ExecutionContextState>();
+            if (!turingComplete)
+            {
+                callFlags &= ~CallFlags.AllowCall;
```

Comment:
We can remove it or throw and exception, what do you prefer @erikzhang ?

Reply:
Exception is good.